sass/directives/_carousels.scss

```diff
@@ -105,7 +105,10 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  text-align: center;
+  line-height: 28px;//this is for vertically centering arrows in ::after
```

Comment:
I kind of need it to exact so that it fits correctly in the div. `em`s are more relative and could throw off alignment.